sunpy/__init__.py

```diff
@@ -48,4 +48,23 @@ class UnsupportedPythonError(Exception):
     # Load user configuration
     config = load_config()
 
+    import logging
+
+    # Use the root logger as a dummy log before initializing Astropy's logger
+    log = logging.getLogger()
```

Comment:
Do we end up initilsing a logger with the name "sunpy"? i.e. so someone can do `logging.getLogger("sunpy")` later to get the logger?

Reply:
This is now fixed and therefore will work.